docs/data/joy/components/select/SelectMultipleFormSubmission.js

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import Button from '@mui/joy/Button';
+import Select from '@mui/joy/Select';
+import Option from '@mui/joy/Option';
+import Stack from '@mui/joy/Stack';
+
+export default function SelectMultipleFormSubmission() {
+  return (
+    <form
+      onSubmit={(event) => {
+        event.preventDefault();
+        const formData = new FormData(event.currentTarget);
+        const formJson = Object.fromEntries(formData.entries());
```

Comment:
It is strange that `formJson.pets` is a string, not an array. Is this expected?

I got `"[\"dog\"\"` instead of `["dog"]`.

Reply:
When you console.log `formJson.pets` it indeed prints `array`, so issue you described maybe have to do something with `alert`

https://github.com/mui/material-ui/assets/60743144/5f9ef638-8829-43f4-a212-14ee3024e924

Reply:
> Is this expected?

Yes, this is expected because if `Select` has `multiple` prop then `value` of `hidden input` is stringified. so users need to do `JSON.parse` to convert it to `array`. 

https://github.com/mui/material-ui/blob/a731e863cb13c83f4d6b9b4c7dd3633fad8f1ed4/packages/mui-base/src/useSelect/useSelect.ts#L51-L57



So I've modified demo and parsed `string` to `array`. Now demo is looking good

Reply:
Got it.